server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailGetMethodContract.scala

```diff
@@ -4343,6 +4343,53 @@ trait EmailGetMethodContract {
          |}""".stripMargin)
   }
 
+  @Test
+  def bodyValuesCallShouldBeImplicit(server: GuiceJamesServer): Unit = {
```

Comment:
is it in the spec also?

Reply:
Oh I have the answer:

```
If the standard properties argument is omitted or null, the following default MUST be used instead of “all” properties:

[ "id", "blobId", "threadId", "mailboxIds", "keywords", "size",
"receivedAt", "messageId", "inReplyTo", "references", "sender", "from",
"to", "cc", "bcc", "replyTo", "subject", "sentAt", "hasAttachment",
"preview", "bodyValues", "textBody", "htmlBody", "attachments" ]
```

Reply:
Removed the commit, that will teach me to do TDD